plugin/context.go

```diff
@@ -7,10 +7,10 @@ package plugin
 // For requests this is built in app/plugin_requests.go
 // For hooks, app.PluginContext() is called.
 type Context struct {
-	SessionId      string
-	RequestId      string
-	IpAddress      string
+	SessionID      string
+	RequestID      string
+	IPAddress      string
```

Comment:
@crspeller - Any concerns with changing these?

Reply:
Yes. It's a breaking change for plugins as they import this package and use these fields. Best to revert.

Reply:
Thanks. @haardikdharma10 - please revert these.

Reply:
Done!